src/components/lotes/LoteForm.jsx

```diff
@@ -109,6 +115,15 @@ class LoteForm extends Component {
     }));
   }
 
+  updateTecnica(obras) {
```

Comment:
I think this isn't a good name for this function. It should be `getTechnicas` or something like that. Update sounds like a DB call